### DISCUSSION TITLE
Estimated date of 0.14 release?

### DISCUSSION THREAD

**Discussion by itamar82** [mirumee/ariadne#720] [General]
Hi!! Looks like a lot of great improvements in master that I'd like to incorporate into my systems. Is there an ETC for when the next release (>0.13.0) will occur?

**Comment by rafalp** [comment]
I don't do estimates, but I'll try to ship release sometime next week.

**Reply by itamar82** [reply]
hah thank you!

**Comment by rafalp** [comment]
Release notes: https://ariadnegraphql.org/blog/2021/11/24/ariadne-0-14-0

![It's done](https://c.tenor.com/caDSgVfhRMcAAAAC/consider-it-done-its-done.gif)

**Reply by gregbrowndev** [reply]
Great work on the release! Thank you 🙏

**Reply by itamar82** [reply]
thank you!
